lib/webdriveragent.js

```diff
@@ -59,6 +61,10 @@ class WebDriverAgent {
 
     this.updatedWDABundleId = args.updatedWDABundleId;
 
+    this.usePreinstalledWDA = args.usePreinstalledWDA;
+    this.preInstalledWDABundleId = args.preInstalledWDABundleId;
```

Comment:
Do we need this capability in general? Can its value be calculated as `${updatedWDABundleId}.xctrunner` instead?

Reply:
Could be with additional a few changes. (At least needed to add if condition in a few places) I initially re-used the capability, but this PR itself does not update bundle id itself. So the naming could be confusing for the behavior.
So i defined a new capability to simplify the implementation and purpose at this time.

Reply:
my point is the less capabilities we have the better it is (easier to maintain)

Reply:
yea, understood this point as well. Then, https://github.com/appium/WebDriverAgent/pull/687/commits/211a10f6b87afe1a2809890a47010a7a3b9e38f0 and https://github.com/appium/appium-xcuitest-driver/pull/1609/commits/d2bfc15684db6cb3f454dab6e03a014e2cbab0c6 helps to work with `updatedWDABundleId`. Then, ignoring xcodebuild instance itself might be safe (rather than adding if condition in xcodebuild instance)

Reply:
Also, I addressed the WDA must have been a package built with over Xcode 12 in https://github.com/appium/appium-xcuitest-driver/commit/d2bfc15684db6cb3f454dab6e03a014e2cbab0c6 for safe to follow `xctrunner` rule. (We do not need to take care the case, but for safe as implementation's requirement with this change)